spec/models/order_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe Order, type: :model do
+  let!(:order) {FactoryBot.create :order}
+  let!(:order_item) {FactoryBot.create :order_item, order_id: order.id}
+  let!(:user) {FactoryBot.create :user}
+
+  describe Order do
```

Comment:
describe, context, it dùng như thế nào e ?

Reply:
describe, context, it để thể hiện hành vi, tổ chức của codes giống như một cuộc hội thoại.
Phương thức describe tạo ra một example group.
Bên trong describe block,  có thể khai báo các nested groups sử dụng describe hoặc context, hoặc  có thể khai báo các examples sử dụng it.

Reply:
khi nào dùng describe, khi nào dùng context em ?

Reply:
Describe để nhóm theo class, module, method (hay theo action của controller).
Context để nhóm các điều kiện của example.